rlog.go

```diff
@@ -357,6 +506,13 @@ func isTrueBoolString(str string) bool {
 // decorates it accordingly and assembles the entire line. It then
 // uses the standard log package to finally output the message.
 func basicLog(logLevel int, traceLevel int, format string, prefixAddition string, a ...interface{}) {
+	// Check if it's time to load updated information from the config file
+	now := time.Now()
+	if settingCheckInterval > 0 && now.Sub(lastConfigFileCheck) > settingCheckInterval {
```

Comment:
Good point. But I think the lock should be in 'initialize', since it possibly updates linked lists and such. Things you don't want multiple people to change at the same time.